docker/pythonsysds.Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN apt-get update -qq \
 	&& mv apache-maven-$MAVEN_VERSION /usr/lib/mvn \
 	&& git clone --depth 1 https://github.com/apache/systemds.git systemds && \
 	cd /usr/src/systemds/ && \
-	mvn clean package -P distribution && \
+	mvn --no-transfer-progress clean package -P distribution && \
```

Comment:
some people find that `-ntp` might mean a combination of `n`, `t`, `p`. Hence using full name of the flag for clarity.